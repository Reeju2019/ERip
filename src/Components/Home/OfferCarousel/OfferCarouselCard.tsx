import { Image } from 'react-bootstrap'

interface Item {
  data: string
}

const OfferCarouselCard: React.FunctionComponent<Item> = (data) => {
  return (
    <>
      <div>
        <Image src={data.data} className='d-block h-100 w-auto' alt={data.data} />
      </div>
    </>
  )
}

export default OfferCarouselCard
