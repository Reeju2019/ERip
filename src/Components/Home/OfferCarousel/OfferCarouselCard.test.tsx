import { render } from '@testing-library/react'
import renderer from 'react-test-renderer'
import ReactDOM from 'react-dom'
import OfferCarouselCard from './OfferCarouselCard'

const data ="a"

test('offer carousel', () => {
  render(<OfferCarouselCard data={data}/>)
})

describe('pnf rendering specification', () => {
  it('SnapShot testing', () => {
    const component = renderer.create(<OfferCarouselCard data={data}/>)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
  it('render without crashing', () => {
    const div = document.createElement('div')
    render(<OfferCarouselCard data={data}/>)
    ReactDOM.unmountComponentAtNode(div)
  })
})
