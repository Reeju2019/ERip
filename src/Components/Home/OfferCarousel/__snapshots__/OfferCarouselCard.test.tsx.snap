// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pnf rendering specification SnapShot testing 1`] = `
<div>
  <img
    alt="a"
    className="d-block h-100 w-auto"
    src="a"
  />
</div>
`;
